.github/ISSUE_TEMPLATE/5.priority-support.yml

```
@@ -0,0 +1,39 @@
+name: 'Priority support: SLA ⏰'
+description: I'm a Premium plan user and I have the priority support add-on, I can't find a solution to my problem with MUI X.
+title: '[question] '
+labels: ['status: needs triage', 'support: commercial', 'support: unknown']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please provide a searchable summary of the issue in the title above ⬆️.
+  - type: checkboxes
+    attributes:
+      label: Duplicates
+      description: Please [search the history](https://github.com/mui/mui-x/issues) to see if an issue already exists for the same problem.
```

```suggestion
      description: Please [search the history](https://github.com/mui/material-ui/issues) to see if an issue already exists for the same problem.
```

Should we ask people to search this repo here? If they came to this repository to create the issue is likely not related to X.

---

Yes, this makes sense.